src/components/Mint/MintPublic.tsx

```diff
@@ -80,6 +81,13 @@ const MintPublic: React.FC<Props> = ({
 				],
 			})
 			onMint()
+		} catch (err: any) {
+			// eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
Does this comment need to be on the same line as the any definition?
If the linter moved it then disable-next-line is a feature